### ISSUE TITLE
How to customize name of log folder?

### ISSUE THREAD

**Issue by YZH0216** [microsoft/RD-Agent#446]
the name of log folder is defined in logger.py line 58 to 61:
    def __init__(self, log_trace_path: Union[str, None] = RD_AGENT_SETTINGS.log_trace_path) -> None:
        if log_trace_path is None:
            timestamp = datetime.now(timezone.utc).strftime("%Y-%m-%d_%H-%M-%S-%f")
            self.log_trace_path = Path.cwd() / "log" / timestamp

and this class is initialized in the __init__.py:
rdagent_logger: RDAgentLog = RDAgentLog()

How can I customize the variable "log_trace_path", if somehow I can define it in ".env" file?

**Comment by XianBW** [comment]
Yes, you can set "LOG_TRACE_PATH" in .env file, but remember to modify it every time you run the program.

Example:
`LOG_TRACE_PATH=./log/1st_test_in_10_21`

If you don't want to modify the .env file every time, you can set this environment variable before running the command, like:
`LOG_TRACE_PATH=./log/1st_test_in_10_21 rdagent fin_model`

**Comment by YZH0216** [comment]
thank you very much!
